test/dlc_tests/eks/pytorch/training/test_eks_pytorch_training.py

```diff
@@ -93,6 +93,57 @@ def test_eks_pytorch_single_node_training(pytorch_training):
         run("kubectl delete pods {}".format(pod_name))
 
 
+@pytest.mark.skipif(not is_pr_context(), reason="Skip this test. It is already tested under PR context and we do not have enough resouces to test it again on mainline pipeline")
+@pytest.mark.model("resnet18")
+def test_eks_pt_s3_single_node_training(pytorch_training):
```

Comment:
Please change the name to include the term `s3_plugin` to make the test name clearer on reports.

Please also add a pytest marker `@pytest.mark.integration("pt_s3_plugin")` or something similar to denote that this is a test for the PT S3 Plugin feature.